guides/source/action_text_overview.md

```diff
@@ -8,149 +8,145 @@ rich text content.
+- Adds the `image_processing` gem for analysis and transformations of the embedded images and other attachments with Active Storage. Please refer to the [Active Storage Overview](active_storage_overview.html) guide for more information about it.
+- Adds migrations to create the following tables that store rich text content and attachments: `action_text_rich_texts`, `active_storage_blobs`, `active_storage_attachments`, `active_storage_variant_records`.
+- Creates `actiontext.css` and imports it into `application.css`. The Trix stylesheet is also included in the `application.css` file.
+- Adds the default view partials `_content.html` and `_blob.html` to render Action Text content and Active Storage attachment (aka blob) respectively.
 
-After the installation is complete, a Rails app should have the following changes:
+Thereafter, executing the migrations will add the new `action_text_*` and `active_storage_*` tables to your app:
 
-1. Both `trix` and `@rails/actiontext` should be required in your JavaScript entrypoint.
+```bash
+$ bin/rails db:migrate
+```
+
+When the Action Text installation creates the `action_text_rich_texts` table, it uses a polymorphic relationship so that multiple models can add rich text attributes. This is done through the `record_type` and `record_id` columns, which store the ClassName of the model, and ID of the record, respectively.
 
-    ```js
-    // application.js
-    import "trix"
-    import "@rails/actiontext"
-    ```
+INFO: With polymorphic associations, a model can belong to more than one other model, on a single association. Read more about it in the [Active Record Associations guide](https://guides.rubyonrails.org/association_basics.html#polymorphic-associations).
 
-2. The `trix` stylesheet will be included together with Action Text styles in your `application.css` file.
+Hence, if your models containing Action Text content use UUID values as identifiers, then all models that use Action Text attributes will need to use UUID values for their unique identifiers. The generated migration for Action Text will also need to be updated to specify `type: :uuid` for the record references line.
+
+```ruby
+ t.uuid "record_id", null: false
+ ```
 
 ## Creating Rich Text Content
 
-Add a rich text field to an existing model:
+This section explores some of the configurations you'll need to follow to create rich text.
+
+The RichText record holds the content produced by the Trix editor in a serialized `body` attribute. It also holds all the references to the embedded files, which are stored using Active Storage. This record is then associated with the Active Record model which desires to have rich text content. The association is made by placing the `has_rich_text` class method in the model that you’d like to add rich text to.
 
 ```ruby
-# app/models/message.rb
-class Message < ApplicationRecord
+# app/models/blog.rb
```

Comment:
Why change it from `Message` to `Blog`? Personally, it doesn't make much sense to me the name of that entity, unless there are many blogs in the sample application. Maybe call it `Article`? As exemplified by the root path in the routes. What do you think?

Reply:
Good point, that can be made clearer. I think `Blog` was in the sense of `BlogPost`, but maybe just `Post` or `Article` as you suggested will be clearer/simpler.

Reply:
Thanks @brunoprietog this is a good suggestion. `Article` is clearer. I'll go ahead and make that change.